index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <script src="https://kit.fontawesome.com/04970764d6.js" crossorigin="anonymous"></script>
+</head>
+<body>
+        <header class="header">
+            <p class="logo">My Logo</p>
```

Comment:
- [ ] [OPTIONAL] Normally a site's logo will enable users to redirect back to the homepage. In that case, it will be nice if you could refactor the `p` tag with `a` tag.